pkg/virt-controller/watch/migration.go

```diff
@@ -619,6 +621,28 @@ func (c *MigrationController) createTargetPod(migration *virtv1.VirtualMachineIn
 		}
 	}
 
+	// The target pod may share resources with the sources pod (RWX disks for example)
+	// Therefore, it needs to share the same SELinux categories to inherit the same permissions
+	// Note: there is a small probablility that the target pod will share the same categories as another pod on its node.
+	//   It is a slight security concern, but not as bad as removing categories on all shared objects for the duration of the migration.
+	if vmi.Status.SelinuxContext == "" {
+		return fmt.Errorf("SELinux context not present on VMI")
```

Comment:
What if we are not running on SELinux enabled node? Should we error out here?

Reply:
Good point, will address.
About the feature gate, please see my initial reply there https://github.com/kubevirt/kubevirt/pull/9246#discussion_r1179471001

Reply:
Done, PTAL!